packages/vue_frontend_rpc/src/assets/css/globalStyles.css

```diff
@@ -90,9 +90,9 @@ span.v-btn__content {
     text-transform: none;
 }
 
-.v-input.v-textarea.v-input--hide-details.v-input--dense.theme--light.v-text-field.v-text-field--is-booted.v-text-field--enclosed.v-text-field--outlined > .v-input__control {
+/* .v-input.v-textarea.v-input--hide-details.v-input--dense.theme--light.v-text-field.v-text-field--is-booted.v-text-field--enclosed.v-text-field--outlined > .v-input__control {
```

Comment:
redundant?

Reply:
seems like. but I prefer to leave it meanwhile as a reference until final ux confirmation